src/site/content/en/progressive-web-apps/progressively-enhance-your-pwa/index.md

```diff
@@ -4,6 +4,7 @@ title: Progressively enhance your Progressive Web App
 subhead: Building for modern browsers and progressively enhancing like it's 2003
```

Comment:
FYI you don't need to actually move the content from the blog directory to the PWA directory. Sorry for the misunderstanding. You just need to add the entry to `_data/paths/progressive-web-apps.js`

Reply:
It's OK this time. I'm just trying to save you some extra unnecessary work.

Reply:
Ah, I see. Thanks for the info.